.github/workflows/ci.yml

```diff
@@ -40,7 +40,7 @@ jobs:
 
           - name: Test
             run: |
-              pytest -v --color=yes tests/
+              pytest -v --color=yes --doctest-modules --doctest-glob='*.md'
```

Comment:
Will this also run the regular tests, in addition to the docs?

Reply:
Yeup!